lib/validators/stale.js

```diff
@@ -13,10 +13,14 @@ class Stale extends Validator {
 
   async validate (context, validationSettings) {
     let days = validationSettings.days || 20
-    let typeSetting = validationSettings.type || ['issues', 'pull_request']
+    let typeSetting = validationSettings.type || ['issue', 'pr']
```

Comment:
just double checking, why put both issues and pr as default. not a deal breaker but if it were me, i would rather have a less is more approach

Reply:
Because you wouldn't be specifying a `stale` validator if you don't intend for it to do something. 
Let's say the config has `stale` but no `type` specified it wouldn't do anything. Why make the user configure more when they just want to be notified of all stale issues and PR?

I would argue that in fact there might be more code if `type` is not specified.  You'd have to add to an error to the result. You'll have a constant and an if statement. 

Unless you want to just do a return when `type` is not specified -- that might be confusing to the user but -- definitely less code.

In short, whichever behavior we want the number of lines of code is not that much different.